File: Selenium/Activities/Activity_11.java
package Activities;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;

public class Activity_11 {
    public static void main(String[] args) {
        WebDriver driver = new ChromeDriver();
        driver.get("https://v1.training-support.net/selenium/dynamic-controls");
        System.out.println("The page title is: " + driver.getTitle());
        System.out.println(driver.findElement(By.xpath("//input[@class='willDisappear']")).isSelected());
        driver.findElement(By.xpath("//input[@class='willDisappear']")).click();
        System.out.println(driver.findElement(By.xpath("//input[@class='willDisappear']")).isSelected());
        driver.quit();

    }
}

File: Selenium/Activities/Activity_11.py
from selenium import webdriver
from selenium.webdriver.common.by import By

with webdriver.Chrome() as driver:
    driver.get("https://v1.training-support.net/selenium/dynamic-controls")
    print("The page title is: ", driver.title)
    print(driver.find_element(By.XPATH, "//input[@class='willDisappear']").is_selected())
    driver.find_element(By.XPATH, "//input[@class='willDisappear']").click()
    print(driver.find_element(By.XPATH, "//input[@class='willDisappear']").is_selected())
    driver.quit()